server/pulp/server/webservices/views/util.py

```diff
@@ -1,15 +1,29 @@
 import functools
 import json
 import sys
+from datetime import datetime
 from functools import wraps
 
 from django.http import HttpResponse
 from django.utils.encoding import iri_to_uri
 
-from pulp.common import error_codes
+from pulp.common import dateutils, error_codes
 from pulp.common.util import decode_unicode, encode_unicode
+from pulp.server.compat import json_util
 from pulp.server.exceptions import PulpCodedValidationException, InputEncodingError
-from pulp.server.webservices.controllers.base import json_encoder as pulp_json_encoder
+
+
+def pulp_json_encoder(thing):
```

Comment:
Consider renaming `thing` to `obj` which is the non-reserved word typically used to represent an object of generic type.